_layouts/base.html.haml

```diff
@@ -189,11 +200,4 @@
           s.parentNode.insertBefore(po, s);
         })();
       </script>
-      <script type="text/javascript">
```

Comment:
As an experiment, I 've moved the Javascript (but not the noscript element) down (against GTM documentation requirements - which don't distinguish between noscript and script) because we have a policy to place all javascripts at the bottom of the page because rendering ("showing something") is a higher priority than reaction.

I did prioritize it to be the first of the javascripts (so even before getting the menu to work etc), as tracking users that leave our page quickly is interesting too.